features/step_definitions/visual_editor_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^The visual editor private beta feature flag is (enabled|disabled)$/) do |enabled|
+  @test_strategy ||= Flipflop::FeatureSet.current.test!
+  @test_strategy.switch!(:govspeak_visual_editor, enabled == "enabled")
+end
+
+When(/^I start creating a new publication$/) do
+  visit new_admin_publication_path
+end
+
+Then(/^I should see the visual editor instead of the govspeak editor$/) do
```

Comment:
The name of this "then" clause would imply to me that we are checking that the govspeak editor is not visible as well. How easy that is in reality I don't know, checking things are not visible is very tricky sometimes

Reply:
Good shout, let me try something.